R/reexports.R

```diff
@@ -0,0 +1,8 @@
+#' @export
```

Comment:
Can we have some brief comments in here on why we are doing - just so we have it in writing.